### ISSUE TITLE
[Accessibility]: Missing audio cue on navigating via Go to Definition

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#35094]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33509

### Problem description

Performing a screen reader "click" on the Go to Definition button in the Code Intel popover potentially takes one to a different file, but the screen reader doesn't announce anything about where we have navigated.

See the attached screen recording.

https://user-images.githubusercontent.com/93103176/167314970-ba50c302-71e7-4cb4-8367-4d695c0e9709.mov

The VoiceOver text doesn't changed, indicating that nothing new was said.

(Note: Control + Option + Space only shows up as Space on the virtual keyboard due to a macOS/VoiceOver bug.)

### Expected behavior

The screen reader should announce the repo name, the file name, and maybe the line number and line text too. The repo name could potentially be omitted if it is the same repo (or it could say~ "this repo"), and the file name could potentially be omitted if we navigated to elsewhere in the same file (or it could say~ "this file").

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by varungandhi-src** [comment]
Right now, I don't know how long fixing this will take, or if it needs input from Design, or if it can be fixed by GitStart, so leaving those unchecked.

**Comment by umpox** [comment]
Fixed through https://github.com/sourcegraph/sourcegraph/pull/35413
